src/lib/pages/backoffice/Item/ItemDetails/ItemCirculation/ItemCirculation.js

```diff
@@ -23,7 +23,10 @@ import {
 
 class ItemStatusMessageOnLoan extends Component {
   render() {
-    const { circulation } = this.props;
+    const { circulation, title } = this.props;
+
+    const emailBody = `${circulation.patron.name}, \n\n "${title}" \n\n\n${invenioConfig.APP.emailSignature}`;
+    const emailSubject = `${invenioConfig.APP.emailSignature} - "${title}"`;
```

Comment:
If used like this, it would be maybe better to have in the config:
```
emailSubjectPrefix: 'InvenioILS',
emailFooter: 'Kind regards,\nInvenioILS'
```
WDYT?